debian/copyright

```diff
@@ -0,0 +1,14 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Contact: Kano developers <dev@kano.me>
+
+Files: *
+Copyright: 2019 Kano Computing Ltd.
+License: GPL-2+
+
+Files: *.png
+Copyright: 2019 Kano Computing Ltd.
+License: All rights reserved
+
+Files: *.wav
+Copyright: 2019 Kano Computing Ltd.
+License: All rights reserved
```

Comment:
Do we need copyrights for pngs and wavs? Are we ever going to be putting these in this repo?